CONTRIBUTING.md

```diff
@@ -41,11 +47,12 @@ If you wish to contribute code (features or bug fixes), see the [Submitting a pa
 
 ### Submitting a patch
 
-Feel free to contribute, we love to receive Pull
-Requests! If you have any questions throughout the process, come chat with us first on [Slack](#where-to-seek-for-help)!
+Feel free to contribute fixes or minor features, we love to receive Pull
+Requests! If you are planning to develop a larger feature, come talk to us
+first on [Slack](#where-to-seek-for-help)!
 
-When contributing, please follow the guidelines in this document and [WRITING-DOCUMENTATION.md](WRITING-DOCUMENTATION.md).
-They will cover topics such as the different Git branches we use, the commit message
+When contributing, please follow the guidelines provided in this document and [WRITING-DOCUMENTATION.md](WRITING-DOCUMENTATION.md). They
```

Comment:
Nitpick: I think somewhere we discussed split lines by sentences.

Reply:
You mean to use semantic line breaks?

I just happened to think about the whole line wrapping situation and pushed a commit removing all artificial line breaks before seeing your comments. Especially for natural language content, artificial line breaks are a pain for editing. I rather think it should be up to a user’s editor preference to set-up soft wrapping for Markdown files to get the reading experience they like.